nf_core/pipeline-template/assets/methods_description_template.yml

```diff
@@ -0,0 +1,25 @@
+id: "${workflow.manifest.name.replace('/', '-')}-methods-description"
+description: "Suggested methods description text of pipeline usage."
+section_name: "${ workflow.manifest.name } Methods Description"
+section_href: "https://github.com/${workflow.manifest.name}"
+plot_type: "html"
+## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
+## You inject any metadata in the Nextflow '${workflow}' object
+data: |
+  <b>Methods</b>
```

Comment:
```suggestion
  <h4>Methods</h4>
```

![download-1](https://user-images.githubusercontent.com/465550/185136667-42af4ce4-379d-4da6-8eef-495756d2f62e.png)

Reply:
Didn't see that, my bad **raps knuckles with ruler**

Reply:
`<b>`, `<br>` and `&nbsp;` should be banned from HTML 😆 

Sorry, can't help myself...